### ISSUE TITLE
Order of items

### ISSUE THREAD

**Issue by saurabh-khanna** [theinvisiblelab/pictopercept#1]
@keelus I would also like you to think a little bit about the order in which we show items (i.e. pairs of pictures) to survey participants. We currently draw pictures randomly from the whole folder of images, but there might be better ways to do this.

For example, if a participant has already compared a middle-aged black male with a young white female, do we gain anything more by showing the exact same combination of features again and again? Something to think careffully about.

**Comment by keelus** [comment]
You're right, relying only on pure randomness may not always be the best option. Maybe we could sort the entire dataset differently instead of shuffling it and picking the first n elements?

I am no pandas/numpy expert, so do you know if it would be possible, knowing the different characteristics that each image has (such as age, etnicity, gender, ...), generate a set of n elements that is "balanced" to show images with different characteristics? Distributed more or less equally (they could have a bit of randomness to vary).
